### ISSUE TITLE
OrderRiskService Deprecated / Obsolete

### ISSUE THREAD

**Issue by czupet** [nozzlegear/ShopifySharp#1073]
Now that the OrderRiskService has been deprecated is there any other option to obtain the same information?

**Comment by clement911** [comment]
Yes you can use this: https://shopify.dev/docs/api/admin-graphql/2024-04/objects/Order#field-order-risk

**Comment by clement911** [comment]
You have to use the `GraphService` to query this API.
